gtest/src/manager.rs

```diff
@@ -1180,7 +1177,7 @@ impl JournalHandler for ExtManager {
         // the queue.
         self.gas_allowance = Gas(GAS_ALLOWANCE);
         self.dispatches.push_front(dispatch);
-        self.block_info.height = self.block_info.height.saturating_add(1);
+        self.blocks_manager.move_blocks_by(1);
```

Comment:
```suggestion
        self.blocks_manager.next_block();
```

Reply:
done in [a10c45e](https://github.com/gear-tech/gear/pull/3999/commits/a10c45e0e5685cf2131f2a99fed219d55dabf4fb)